stdlib/typing.pyi

```diff
@@ -428,17 +428,16 @@ class Generator(Iterator[_YieldT_co], Generic[_YieldT_co, _SendT_contra, _Return
     @property
     def gi_yieldfrom(self) -> Generator[Any, Any, Any] | None: ...
 
-# NOTE: Technically we would like this to be able to accept a second parameter as well, just
-#   like it's counterpart in contextlib, however `typing._SpecialGenericAlias` enforces the
-#   correct number of arguments at runtime, so we would be hiding runtime errors.
-@runtime_checkable
-class ContextManager(AbstractContextManager[_T_co, bool | None], Protocol[_T_co]): ...
+# NOTE: Prior to Python 3.13 these aliases are lacking the second _ExitT_co parameter
+if sys.version_info >= (3, 13):
+    ContextManager = AbstractContextManager
+    AsyncContextManager = AbstractAsyncContextManager
```

Comment:
Can we do it like this instead?

```suggestion
    from contextlib import AbstractContextManager as ContextManager
    from contextlib import AbstractAsyncContextManager as AsyncContextManager
```
(With some corresponding changes elsewhere.)

That way, these look like the same type, not just an alias.

Reply:
They were defined as simple assignments before the change as well: https://github.com/python/typeshed/pull/11048/files#diff-b12c4be0f8e39f89957bbe7be6b6327185e18b69ea80a0f62c0746bb26f4a58e

I'm not quite sure why, it's possible that the original names are used elsewhere in `typing.pyi` but I can reorganize things to turn these aliases into re-exports, if you'd prefer that.